lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -388,9 +388,9 @@ def add_level2_tax(xml, options = {})
       def add_level3_tax(xml, options = {})
         if (level3 = options[:level_3_data])
           xml.tag! :PC3VATtaxAmt, byte_limit(level3[:vat_tax], 12) if level3[:vat_tax]
-          xml.tag! :PC3AltTaxAmt, byte_limit(level3[:alt_tax], 9) if level3[:alt_tax]
           xml.tag! :PC3VATtaxRate, byte_limit(level3[:vat_rate], 4) if level3[:vat_rate]
           xml.tag! :PC3AltTaxInd, byte_limit(level3[:alt_ind], 15) if level3[:alt_ind]
+          xml.tag! :PC3AltTaxAmt, byte_limit(level3[:alt_tax], 9) if level3[:alt_tax]
         end
```

Comment:
Purely out of curiosity -- was there a reason this field got reordered?

Reply:
I realized that we were sending the NewOrder request incorrectly - it was passing the `PC3AltTaxAmt` tag before the `PC3VATtaxRate` and `PC3AltTaxInd` tags which according to the Orbital spec is wrong